test/functional/repository/virtual-column/entity/utils.ts

```diff
@@ -0,0 +1,30 @@
+import { DatabaseType } from "../../../../../src";
+
+export function concat(type: DatabaseType, ...strings: string[]) {
+    switch (type) {
+        case "postgres":
+        case "oracle":
+        case "sqlite":
+            return strings.join("||");
+        case "mysql":
+        case "mariadb":
+            return "concat(" + strings.join(",") + ")";
+        default:
+            throw new Error(`dialect '${type}' still not implement 'concat()'`);
```

Comment:
Adding new driver would require changes in this test file?

Reply:
Don't require, it still pass test.